bench/config/procfile.py

```diff
@@ -4,12 +4,11 @@
 import click
 
 import bench
-from bench.app import use_rq
 from bench.bench import Bench
 from bench.utils import which
 
 
-def setup_procfile(bench_path, yes=False, skip_redis=False):
+def setup_procfile(bench_path, yes=False, skip_redis=False, skip_web=False, skip_watch=os.environ.get("CI"), skip_socketio=False, skip_schedule=False, with_coverage=False):
```

Comment:
Can we set `None` for skip_watch and then set os.environ.get() inside? Have had a weird experience setting default values like this (might not affect this specific case because its a CLI that would be freshly imported/called everytime, but may as well follow that pattern everywhere)

Reply:
https://github.com/frappe/bench/compare/95182c89ed62720628dd97e1e1a4002f9bbdcec2..786b63def57e582d508917a6ca5d13a11be1a8b3